### PR TITLE
Remove wercker, use sd to publish

### PR DESCRIPTION
**Pull request by d2lam**
Use screwdriver instead :d2celebrate:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Screwdriver Datastore Dynamodb
-[![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][wercker-image]][wercker-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
+[![Version][npm-image]][npm-url] ![Downloads][downloads-image][![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
```
Comment (petey) [inline README.md]: Now there is no build status badge? 

status

**Review by petey — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (d2lam) [reply README.md]: Fixed

**Review by petey — APPROVED** [review]

**Comment by petey** [comment]
👍

**Review by minzcmu — APPROVED** [review]
👍

**Pull request merged**